### PR TITLE
Make Relationship nullable

### PR DESCRIPTION
**Pull request by rpanman-sonatype**
This PR allows Relationships to be made nullable as discussed in #11.

### CODE REVIEW

**Comment by rpanman-sonatype** [comment]
I wasn't sure how to complete the CHANGELOG so that will probably need some work.

File: README.rst [hunk 1]
```diff
@@ -91,6 +91,17 @@ Relationships can also be declared:
        owner = attributes.Relationship('Owner', related_model=User)
        contacts = attributes.Relationship('Contacts', related_model=User, many=True)
 
+Relationships can also be nullable if needed - an indication that this could be needed is
```
Comment (barakalon) [inline README.rst]: I don't think this needs its own section in the README. 
Maybe just add "nullable=True" to the example above?

**Review by barakalon — COMMENTED** [review]
Sweet! One small comment on the README.

Thanks for the change!

File: README.rst (hunk 1 above)
Comment (rpanman-sonatype) [reply README.rst]: Change made - if you could check and resolve the confirmation if you're happy with it 😄 

Please also let me know if you'd like me to squash the commits.

**Review by barakalon — APPROVED** [review]
Squashing is nice :)

**Review by barakalon — APPROVED** [review]

**Comment by barakalon** [comment]
Might need @RookieRick to help us merge/deploy

**Comment by rpanman-sonatype** [comment]
I've squashed it 😄

**Comment by rpanman-sonatype** [comment]
Hi Guys, any ETA on a merge/release? I think my squash killed the approval as well. Cheers!

**Comment by barakalon** [comment]
I'm not sure who's responsible for this at PlanGrid anymore.

@airstandley?

**Comment by airstandley** [comment]
Ah! Sorry for the long wait, I missed this notification. 

@barakalon The honest answer is that I'm not sure anyone is responsible for this at this time. Let me ask around and find out next steps. 

In the mean time I will try to get this merged and a new version released.

**Review by airstandley — APPROVED** [review]

**Comment by barakalon** [comment]
Maybe transfer ownership to me?

**Comment by airstandley** [comment]
>Maybe transfer ownership to me?

That would be the best outcome in my mind 😄

**Comment by airstandley** [comment]
@rpanman-sonatype New version should now be available https://pypi.org/project/soql/1.2.0/

Thank you for the contribution, and again I'm very sorry for the long wait!

**Comment by RookieRick** [comment]
> Might need @RookieRick to help us merge/deploy

Oh wow. Likewise, my apologies. Caught me when I was in the midst of moving (which I just finished doing AGAIN, but this time back out of RV and into a new - to us at least - house)!  <3

**Comment by rpanman-sonatype** [comment]
Thanks and no worries about the delay, I just subclassed it locally so it didn't impact me 😄 - at least I can delete that now and make it pure soql!

Also well done to @barakalon for taking on the mantle of responsibility - maintainers are had to find 🥇

**Pull request merged**